pymatgen/io/vasp/inputs.py

```diff
@@ -672,9 +672,10 @@ def set_temperature(self, temperature: float) -> None:
             temperature (float): Temperature in Kelvin.
         """
         # mean 0 variance 1
```

Comment:
This comment seems incorrect, `np.random.randn` (and the new Generator approach) generate "random floats in the half-open interval [0.0, 1.0).", which is not standard normal distribution.